packages/api/src/service/validity-check.service.ts

```diff
@@ -64,11 +60,11 @@ class ValidityCheckService {
     );
   }
 
-  public deleteAll(): Observable<DeletedData[]> {
+  public deleteAll(): Observable<Number> {
```

Comment:
eslint/tslint prefer 
```suggestion
  public deleteAll(): Observable<number> {
```